joomla/joomla-cms
# Close Button on iOs doesn´t work - Add cursor type to alert to make the close icon working on iOs

Close Button on iOs doesn´t work

Bug in Bootstrap: Alerts are not closeable on iOs
### Summary of Changes

Added cursor:pointer; to alerts.less and bootstrap_rtl.less to have the altert message be close-able on iOs too.
### Testing Instructions

Please test on iOs (not Chrome emulation or something else)
Before applying the patch:
Type in wrong login credentials using Joomla! Protostar Template.
Try to dismiss the alert by tapping on the close button.
- Nothing will happen

After applying the patch:
Login again with wrong credentials
Try to dismiss the altert
Alert should close now.
### Documentation Changes Required

Sorry i deleted template.css completely and compiled over Template Manager, any better way to do that? Thanks!

![image](https://cloud.githubusercontent.com/assets/828371/18250769/73c28c52-7385-11e6-8bb4-b2a4a394f5a7.png)
